tests/PyomoTest.py

```diff
@@ -83,24 +84,25 @@ def build_generator(size, lifetime, dispatch):
 
   ## fixed OM
   alpha = -10.0 # $10/y for upkeep of a 1 MW generator
-  fixed_om = createRecurringYearly(generator, alpha, size)
+  fixed_om = createRecurringYearly(generator, alpha, projectlife, size)
   cfs.append(fixed_om)
 
   ## variable OM
   alpha = -1.0 # $0.10 per MWh produced
-  var_om = createRecurringHourly(generator, alpha, dispatch)
+  var_om = createRecurringHourly(generator, alpha, projectlife, dispatch)
   cfs.append(var_om)
 
   generator.addCashflows(cfs)
   return generator
 
 
-def build_market(size, life, prices, dispatch):
+def build_market(size, life, prices, projectlife, dispatch):
   """
     Constructs the cash flow of each applicable components
     @ In, size, pyomo.core.base.var, build size
     @ In, life, int, life time of the years to evaluate
     @ In, prices, numpy array, values of pricing
+    @ In, projectlife, int, length of project
```

Comment:
What's the purpose of changing this functionality when previously we were able to get `projectlife` by running `comp.getLifetime()`?